src/wymeditor/plugins/table/jquery.wymeditor.table.js

```diff
@@ -394,13 +410,13 @@ TableEditor.prototype.mergeRow = function () {
     // Just use the td and th nodes
     cells = jQuery(nodes).filter('td,th');
     if (cells.length === 0) {
-        return false;
+        return;
```

Comment:
So there is no need to return `false` here or at the end. Ditto for all the manipulation methods here.